### PR TITLE
Zeek v3.2.0-dev-551-gfda8b98ac for macOS, Linux, and Windows

### LINKED ISSUE #10: Windows Zeek artifact packaging
We now have the pipeline to create a Cygwin-based Zeek binary as described at https://github.com/brimsec/zeek/blob/master/brim/windows/README.md. We still need a zipfile artifact that has the expected structure as the MacOS artifact, and a script/batch file that handles the `ZEEK*` environment variables.

@alfred-landrum already merged https://github.com/brimsec/zeek/pull/9 that's related to this. There will be additional PR’s to automatically build & put that utility into the windows Zeek ZIP artifact.

### PR DESCRIPTION
**Pull request by nwt**
1. Update Zeek to v3.2.0-dev-551-gfda8b98ac. 
1. Make it build with MinGW tools. The resulting Windows executable is missing a lot of functionality but can process pcaps.
1. Update zeekrunner.go for MingGW Zeek.
1. Add a Windows artifact to Brim releases.
1. Advance the macOS deployment target to 10.14 (from 10.10) for the sake of a C++ feature I can't recall.
1. Run the Brim GitHub workflow (renamed to brim.yml from brim-release.yml) for PRs, master, and release tags, and upload the artifacts.

This should close #10, #14, and #24.

### CODE REVIEW

**Comment by henridf** [comment]
> Update Zeek to v3.2.0-dev-551-gfda8b98ac.

Is it possible to add these commits on a separate PR? Also, what does v3.2.0-dev-551-gfda8b98ac refer to? I see the v3.2.0-dev branch in upstream, but no commit gfda8b98ac or reference to "551".

**Comment by nwt** [comment]
@henridf: v3.2.0-dev-551-gfda8b98ac is "git describe" output for fda8b98ac, which is 551 commits past tag v3.2.0-dev. (That was zeek/zeek@master at the time I started merging this branch into brimsec/zeek@master).

Should I simply push the merge commit (c62b41862103eab11d4ec9bd12c1255bb7f54d85) to master so this PR shrinks to the last six commits? Or would you like to see a PR with the merge commit plus minimal necessary changes to brim/ and .github/workflows/brim-release.yml (which I don't have on hand but could prepare)? Or something else?

**Comment by henridf** [comment]
> Should I simply push the merge commit (c62b418) to master so this PR shrinks to the last six commits? 

Hmm. I guess that (assuming this PR isn't squashed upon merging), then the resulting history in master would be the same if you push that merge commit vs doing everything here.

> Or would you like to see a PR with the merge commit plus minimal necessary changes to brim/ and .github/workflows/brim-release.yml (which I don't have on hand but could prepare)? 

This doesn't seem indispensable to me, and it's extra work, so I'd say not worth it.

Overall my concern was just about future readability and maintenance, but after some more thought I think this approach (with an unsquashed merge) is good. 

(Also I was anticpating the "how do we maintain these changes on top of zeek over time", but given that the goal is for your changes to be upstreamed, this shouldn't be an issue).

**Comment by nwt** [comment]
> Overall my concern was just about future readability and maintenance, but after some more thought I think this approach (with an unsquashed merge) is good.

👍 I was planning to merge this by adding a merge commit.

File: .github/workflows/brim-release.yml [hunk 1]
```diff
@@ -5,15 +5,71 @@ on:
...
+
+      # Remove DLLs that break Zeek's OPENSSL_CORRECT_VERSION_NUMBER
+      # CMake test by shadowing DLLs in /mingw64/bin.
+      - if: startsWith(matrix.os, 'windows-')
+        run: rm $SYSTEMROOT/system32/libcrypto-1_1-x64.dll $SYSTEMROOT/system32/libssl-1_1-x64.dll
+
+      - if: startsWith(matrix.os, 'windows-')
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14.3
```
Comment (alfred-landrum) [inline .github/workflows/brim-release.yml]: We're using `go-version: 1.13` in the zq build, can we use that here for consistency?

File: brim/release [hunk 2]
```diff
@@ -1,45 +1,64 @@
 #!/bin/sh -ex
 
-case $(uname -s) in
+install_libpcap() {
```
Comment (alfred-landrum) [inline brim/release:3]: can you add the comment stating why we do this from the removed text ("Compile a recent libpcap, ...")

Comment (nwt) [reply brim/release:3]: @alfred-landrum: That comment is still present at https://github.com/brimsec/zeek/blob/windows/brim/release#L22-L23.

File: .github/workflows/brim-release.yml (hunk 1 above)
Comment (nwt) [reply .github/workflows/brim-release.yml]: @alfred-landrum: Done in 04df34d73039313c781c6664df852c4fa746422b.

**Comment by nwt** [comment]
We don't need this any longer, so I removed it from .github/workflows/brim-release.yml in 04df34d73039313c781c6664df852c4fa746422b.
```yaml
      # Remove this step when the hosted runners have
      # https://github.com/actions/virtual-environments/pull/632.
      - if: startsWith(matrix.os, 'windows-')
        name: Windows MSYS2 installation
        shell: pwsh
        run: |
          Set-PSDebug -Trace 1
          if (Test-Path C:/msys64) { exit 0 }
          $url = 'http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20190524.tar.xz'
          bash -c "curl -LSs $url | tar -xJf - -C /c"
          $env:Path = "C:\msys64\usr\bin"
          # packman-key is a script, so PowerShell can't run it directly.
          bash -c 'pacman-key --init'
          bash -c 'pacman-key --populate msys2'
          # pacman-key starts gpg-agent, causing breakage if pacman
          # upgrades msys-2.0.dll.
          Stop-Process -Name gpg-agent
          # First update pacman to avoid
          # https://github.com/msys2/MSYS2-packages/issues/1960.
          pacman -Sy --noconfirm pacman
          pacman -Syuu --noconfirm
          pacman -Syuu --noconfirm
```

**Pull request merged**